packages/js-commons/src/core/local-files.ts

```diff
@@ -158,6 +173,89 @@ export const getLocalApplicationTransporter = (
       await setGlobalMetadata(appsMetaData);
       return appData;
     },
+
+    storeAsset: async (
+      appId: string,
+      fileName: string,
+      fileData: string | Buffer,
+      isFont = false,
+      fontFamily?: string,
+      weight?: number,
+      fontStyle?: string,
+      fontType?: string,
```

Comment:
Combine this into an optional object property, e.g. `font?: { family, weight, etc..}`